product-preview-card-component-main/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/1e0ef04d8c.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+  <title>PPCC</title>
+</head>
+<body>
+  <section class="container">
+    <div class="content">
+      <img class="container-img" src="./images/mobile.jpg" alt="Perfumes">
+      <div class="inner_cont">
+        <h4>PERFUME</h4>
+      <h1>Gabrielle Essence Eau De Parfum</h1>
+      <p>A floral, solar and voluptuous interpretation composed by Olivier Polge, Perfumer-Creator for the House of CHANEL.</p>
+      <div class="price">
+        <h3>$149.99</h3>
+        <h6>$169.99</h6>
```

Comment:
I see that you used the h1-h6 tags for their styling instead of relying on them for semantic reasons. [Here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) is an excellent resource explaining how you can best use these tags.